app/src/org/commcare/connect/ConnectIdWorkflows.java

```diff
@@ -219,9 +219,9 @@ private static void continueWorkflow() {
             case CONNECT_UNLOCK_BIOMETRIC -> {
```

Comment:
Looks like this task should go as well. It's the last one linking to ConnectIdBiometricUnlockActivity. Anywhere that used this should perform the unlock directly instead, and once this is gone we can also get rid of the obsolete activity.